test/crypto.js

```diff
@@ -0,0 +1,15 @@
+import 'babel-register'
```

Comment:
Why babel-register? We have babel-core installed, you should be able to use import 'babel-core/register'